Documentation/RegistryDraft.md

```diff
@@ -663,15 +663,22 @@ A client MAY use this information
 to determine its preferred strategy for downloading.
 
 A server that indexes but doesn't host packages
-SHOULD respond with a status code of `303` (See Other)
+MAY respond with a status code of `303` (See Other)
 and redirect to a hosted package archive if one is available.
 
 ```http
 HTTP/1.1 303 See Other
 Content-Version: 1
+Content-Length: 2048
+Digest: sha-256=a2ac54cf25fbc1ad0028f03f0aa4b96833b83bb05a14e510892bb27dea4dc812
```

Comment:
@mattt @robertlacroix @yim-lee and myself discussed this more, and I think @robertlacroix preference is for this to be part of the metadata as he originally proposed. any strong reason why we should not move it there?

another good observation @robertlacroix made after discussing the redirecting proxies is that all APIs should support 30x return code. any reason why we should not make this explicit in the spec?